aepsych/plotting.py

```diff
@@ -366,14 +406,14 @@ def plot_strat_3d(
     """Creates a plot of a 2d slice of a 3D strategy, showing the estimated model or probability response and contours
     Args:
         strat (Strategy): Strategy object to be plotted. Must have a dimensionality of 3.
-        parnames (str list): list of the parameter names
+        parnames (List[str], optional): list of the parameter names
```

Comment:
Missing default for outcome label, also optionals should still somewhat describe what will happen if its none if something does happen.